### DISCUSSION TITLE
generating weird answers

### DISCUSSION THREAD

**Discussion by rpking1107** [LostRuins/koboldcpp#60] [Q&A]
first of all, thanks a lot for the amazing project.  My GPU is 3060 12gb and cant run the 13b model, viand somehow oobabooga doesnt work on my CPU, Then i found this project, its so conveinent and easy to deploy!    

But when i tried this on my computer i got really weird answers from models (also quite slow) 
  The model im using is this one:   https://huggingface.co/anon8231489123/gpt4-x-alpaca-13b-native-4bit-128g/tree/main/gpt4-x-alpaca-13b-ggml-q4_1-from-gptq-4bit-128g.     and here are the running time and response i got, these are clearly wrong, did i do anything wrong here? thanks!
![image](https://user-images.githubusercontent.com/17896813/232028255-bbdd8e31-b55e-4bad-936b-91e190f644b3.png)
![image](https://user-images.githubusercontent.com/17896813/232028306-1b1ad43a-a39f-4a57-b3ab-fc916894ff74.png)
![image](https://user-images.githubusercontent.com/17896813/232028549-1ca5124c-d5ae-46f5-891d-1c0fe8ec1480.png)

**Comment by LostRuins** [comment]
Try formatting the prompt in the Alpaca format, or use the Instruct mode from the UI.
